src/libraries/Microsoft.Extensions.Options/src/OptionsCache.cs

```diff
@@ -65,7 +65,7 @@ internal TOptions GetOrAdd<TArg>(string? name, Func<string, TArg, TOptions> crea
 #if NET || NETSTANDARD2_1
             return _cache.GetOrAdd(
                 name ?? Options.DefaultName,
-                static (name, arg) => new Lazy<TOptions>(arg.createOptions(name, arg.factoryArgument)), (createOptions, factoryArgument)).Value;
+                static (name, arg) => new Lazy<TOptions>(() => arg.createOptions(name, arg.factoryArgument)), (createOptions, factoryArgument)).Value;
```

Comment:
We need to add library servicing changes here, right?

Reply:
Do you mean `<ServicingVersion>1</ServicingVersion>`? If yes, I am adding all needed elements to the project file now.

Reply:
Yes, see https://github.com/dotnet/runtime/blob/main/docs/project/library-servicing.md

Reply:
Done.